### PR TITLE
git-api fix

### PR DESCRIPTION
**Pull request by ivanvolkoff**
- added  extensions folder to git-api 
- fixed lack of .jar file in a grouping for javascript-all.jar

### CODE REVIEW

File: api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/modules.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "description": "Dirigible Git Module",
+    "isPackageDescription": true,
+    "dtsPath": "git/extensions/git.d.ts"
+  },
+  {
+    "name": "git/v4/client",
+    "description": "Client API",
+    "api": "client",
+    "versionedPaths": [
+      "git/v3/client",
```
Comment (ThuF) [inline api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/modules.json:13]: I think we don't have `git/v3/client`?

File: api/api-javascript/api-git/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/target
+/bin/
```
Comment (ThuF) [inline api/api-javascript/api-git/.gitignore:2]: Not needed, could you delete it?

File: api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/modules.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "@dirigible/git",
+    "description": "Dirigible Git Module",
+    "isPackageDescription": true,
+    "dtsPath": "git/extensions/git.d.ts"
+  },
+  {
+    "name": "git/v4/client",
+    "description": "Client API",
```
Comment (ThuF) [inline api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/modules.json:10]: `Client API` -> `Git Client API`?

File: api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/git.d.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+declare module "@dirigible/git" {
+    module client {
+
+
+    }
+}
```
Comment (ThuF) [inline api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/git.d.ts:6]: I guess the `d.ts` file content would be added at later point?

**Review by ThuF — CHANGES_REQUESTED** [review]
Few minor comments to be addressed first

File: api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/git.d.ts (hunk 4 above)
Comment (ivanvolkoff) [reply api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/git.d.ts:6]: yes

File: api/api-javascript/api-git/.gitignore (hunk 2 above)
Comment (ivanvolkoff) [reply api/api-javascript/api-git/.gitignore:2]: +1

**Review by ThuF — APPROVED** [review]
+1

**Pull request merged**